pgoapi/pgoapi.py

```diff
@@ -561,7 +561,8 @@ def attempt_catch(self, encounter_id, spawn_point_id, capture_probability=None):
                     self.log.info("Could not feed the Pokemon. (%s)", r)
 
             pokeball = self.inventory.take_next_ball(capture_probability)
-            self.log.info("Attempting catch with ball type {0}  at {1:.2f} % chance. Try Number: {2}".format(pokeball,
+            ball_name = {1: "a Poke Ball", 2: "a Great Ball", 3: "an Ultra Ball", -1: "Nothing"}
```

Comment:
@joychugh Thank you. Fixed in 33377adce658a91b206c6282b784df51459db77c. For some reason I thought they weren't the same IDs.